roles/ssh_hardening/meta/argument_specs.yml

```diff
@@ -0,0 +1,351 @@
+        default: ['22']
+        type: list
+        description: ports on which ssh-server should listen.
+      ssh_client_port:
+        default: '22'
+        description: Specifies the port number to connect on the remote host.
+      ssh_listen_to:
+        default: [0.0.0.0]
+        type: list
+        description: one or more ip addresses, to which ssh-server should listen to.
+          Default i all IPv4 addresses, but should be configured to specific addresses
+          for security reasons
+      ssh_host_key_files:
+        default: []
+        type: list
+        description: Host keys for sshd. If empty ['/etc/ssh/ssh_host_rsa_key', '/etc/ssh/ssh_host_ecdsa_key',
+          '/etc/ssh/ssh_host_ed25519_key'] will be used, as far as supported by the
+          installed sshd version.
+      ssh_host_rsa_key_size:
+        default: 4096
+        description: Specifies the number of bits in the private host RSA key to create.
+      ssh_host_key_algorithms:
+        default: []
+        type: list
+        description: Host key algorithms that the server offers. If empty the default
+          lst will be used. Otherwise overrides the setting with specified list of
+          algorithms. Check `man sshd_config`, `ssh -Q HostKeyAlgorithms` or other
+          sources for supported algorithms - make sure you check the correct version
+      ssh_client_host_key_algorithms:
+        default: []
+        type: list
+        description: Specifies the host key algorithms that the client wants to use
+          in order of preference. If empty the default lst will be used. Otherwise
+          overrides the setting with specified list of algorithms. Check `man ssh_config`,
+          `ssh -Q HostKeyAlgorithms` or other sources for supported algorithms - make
+          sure you check the correct version!
+      ssh_client_alive_interval:
+        default: 600
+        description: specifies an interval for sending keepalive messages.
+      ssh_client_alive_count:
+        default: 3
+        description: Defines the number of acceptable unanswered client alive messages
+          before disconnecting clients.
+      ssh_permit_tunnel:
+        default: false
+        type: bool
+        description: true if SSH Port Tunneling is required.
+      ssh_remote_hosts:
+        default: []
+        type: list
+        description: one or more hosts and their custom options for the ssh-client.
+          Default i empty. See examples in `defaults/ain.yml`
+      ssh_permit_root_login:
+        default: 'no'
+        description: Disable root-login. Set to `'without-password'` or `'yes'` to
+          enable root-login - The quotes are required!
+      ssh_allow_tcp_forwarding:
+        default: 'no'
+        description: "'no' or `False` to disable TCP Forwarding. Set to `'yes'` or\
+          \ `True` to allow TCP Forwarding. If you are using OpenSSH >= 6.2 version,\
+          \ you can specify `'yes'`, `'no'`, `'all'`, `'local'`or`'remote'`."
```

Comment:
why all these `\`?

Reply:
another automatic linting fix... I'll change it.